terra-app-helm/aou-rstudio-chart/templates/persistence.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.persistence.enabled -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ include "app.fullname" . }}-pv
+spec:
+  storageClassName: "{{ .Values.persistence.storageClass }}"
+  capacity:
+    storage: {{ .Values.persistence.size }}
+  accessModes:
+    - {{ .Values.persistence.accessMode }}
+  {{- if .Values.persistence.hostPath }}
+  hostPath:
+    path: {{ .Values.persistence.hostPath }}
+  {{- end }}
+  claimRef:
+    namespace: {{ $.Release.Namespace }}
+    name: {{ include "app.fullname" . }}-pvc
+  {{- if .Values.persistence.gcePersistentDisk }}
+  gcePersistentDisk:
+    pdName: {{ .Values.persistence.gcePersistentDisk }}
+    fsType: ext4
+  {{- end }}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "app.fullname" . }}-pvc
+spec:
+  storageClassName: "{{ .Values.persistence.storageClass }}"
+  volumeName: {{ include "app.fullname" . }}-pv
+  accessModes:
+    - {{ .Values.persistence.accessMode }}
```

Comment:
Just noting a lot of this is highly templated which makes it a bit confusing.. consider removing templates and just putting in raw values where it makes sense. E.g. RStudio probably always wants `accessMode: ReadWriteMany`, `storageClass: default`, etc